[WIP}OSDOCS-8651: Edits from content audit (#73505)
openshift/openshift-docs: modules/nw-dns-operator-status.adoc

:_mod-docs-content-type: PROCEDURE
[id="nw-dns-operator-status_{context}"]
= Checking DNS Operator status

You can inspect the status and view the details of the DNS Operator
using the `oc describe` command.

.Procedure

> . View the status of the DNS Operator:

Reviewer: ```suggestion
* View the status of the DNS Operator:
```

+
[source,terminal]
----
$ oc describe clusteroperators/dns
----
+

> Though the messages and spelling might vary from release to release, the expected status output looks like:

Reviewer: ```suggestion
Though the messages and spelling might vary in a specific release, the expected status output looks like:
```
